commodore/cli.py

```diff
@@ -533,7 +557,9 @@ def package_new(
     to configure golden tests, and the licensing details.
     """
     config.update_verbosity(verbose)
-    t = PackageTemplater(config, slug, name=name, output_dir=output_dir)
+    t = PackageTemplater(
+        config, template_url, template_version, slug, name=name, output_dir=output_dir
+    )
```

Comment:
The template url and version is now set twice, in the constructor and 4 lines below